flowetl/etl/etl/production_task_callables.py

```diff
@@ -8,7 +8,60 @@
 """
 import logging
 
-from airflow.models import DagRun
+from pathlib import Path
+
+from airflow.models import DagRun, BaseOperator
+from airflow.hooks.dbapi_hook import DbApiHook
+
+
+def render_sql_callable(
```

Comment:
Given that this doesn't just render the sql but also runs it, would be good to clarify this in the name of the callable.
```suggestion
def render_and_run_sql_callable(
```

Btw, some of the names of these functions sound a bit confusing with the `_callable` suffix. I'm wondering whether we should just do away with the suffix, or alternatively use two underscores (`...__callable`) to clarify that we're not rendering and running a sql callable here.

Reply:
Have changed to `...__callable`